style.css

```diff
@@ -487,24 +518,25 @@ textarea:focus {
 }
 
 .form-button {
-  margin: 20px 18px 64px 14px;
+  width: 100%;
+  margin-bottom: 15%;
```

Comment:
This `<button>` should be at max 50% of the curent width, kindly fix that 🥳 

<details><summary>Also, to make it centered you probably will need to change the margin a little bit, click here if you need a hint</summary>
    <pre>
      try: `margin: 0 auto 15%;` 😜 
    </pre>
   </details>